deployment/docker-compose/README.md

```diff
@@ -0,0 +1,167 @@
+# Docker Compose Deployment
+
+The CCNP can be deployed in the TDVM by docker compose.
+
+## 1. Prerequisite
+
+### 1.1 Host Side 
+
+The host should enable TDX, and setup service PCCS & service QGS which help generate TDX quote in the guest.
+
+### 1.2 Guest Side
+
+The TDX device in the guest has different names in certain versions
```

Comment:
It may be better to ask users to check device node using "ls /dev/tdx*" and decide which one to use. No need to combine it with TDX 1.0 or TDX 1.5